examples/job.json

```diff
@@ -0,0 +1,25 @@
+{
+    "tempered_dist_family": "boltzmann",
+    "probability_definition": "wedon'tneedthat",
+    "max_replicas": 8,
+    "initial_number_of_replicas": 4,
```

Comment:
Nice, this is the same `job.json` that I've been using for local testing with the controller. The only difference being that I have `initial_number_of_replicas` of 5.